Mathlib/Data/Opposite.lean

```diff
@@ -59,6 +59,10 @@ theorem op_injective : Function.Injective (op : α → αᵒᵖ) := fun _ _ => c
 theorem unop_injective : Function.Injective (unop : αᵒᵖ → α) := fun ⟨_⟩⟨_⟩ => by simp
 #align opposite.unop_injective Opposite.unop_injective
 
+theorem op_surjective : Function.Surjective (op : α → αᵒᵖ) := fun x => ⟨unop x, rfl⟩
```

Comment:
If you put this on line 95, the proof is `equivToOpposite.surjective`

Reply:
I think there's still a weak reason relating to defeq that justifies it existing.

Reply:
Rewrote both, the other one via `symm`, hope that's fine